stubs/pyinstaller/PyInstaller/utils/hooks/__init__.pyi

```diff
@@ -0,0 +1,79 @@
+# https://pyinstaller.org/en/stable/hooks.html
+
+import os
+from _typeshed import AnyOrLiteralStr, StrOrBytesPath, SupportsKeysAndGetItem
+from collections.abc import Callable, Iterable
+from typing import Any, AnyStr
+from typing_extensions import Literal, TypeAlias
+
+import pkg_resources
+from PyInstaller import HOMEPATH as HOMEPATH, isolated
+from PyInstaller.depend.imphookapi import PostGraphAPI
+from PyInstaller.utils.hooks import conda as conda_support
+from PyInstaller.utils.hooks.win32 import get_pywin32_module_file_attribute as get_pywin32_module_file_attribute
+
+_Environ: TypeAlias = SupportsKeysAndGetItem[str, str] | Iterable[tuple[str, str]] | os._Environ[str]
```

Comment:
Ultimately it looks like the `env` arguments are passed to `__exec_python_cmd`, and `__exec_python_cmd` uses it to update a deepcopy of `os.environ`. So I think any mapping will do here, not just instances of `os._Environ` (which are pretty hard to come by :)

https://github.com/pyinstaller/pyinstaller/blob/36aa18acf2a04a4f28e45f9e805480f0df0c577a/PyInstaller/utils/hooks/__init__.py#L55-L59

```suggestion
_Environ: TypeAlias = SupportsKeysAndGetItem[str, str] | Iterable[tuple[str, str]] | Mapping[str, str]
```

Reply:
Is there an easier/shorter way to define "anything that can be used to update a `dict[str, str]` ?

Reply:
Sadly no, not that I'm aware of :(